src/Resources/app/administration/src/module/sw-dashboard/component/sw-dashboard-statistics-promotion-card/sw-dashboard-statistics-promotion-card.spec.js

```diff
@@ -0,0 +1,146 @@
+import { mount } from '@vue/test-utils';
+
+const STATISTICS_APP_NAME = 'StatisticsService';
+
+Shopware.Component.register(
+    'sw-dashboard-statistics-promotion-card',
+    () => import('SwagExtensionStore/module/sw-dashboard/component/sw-dashboard-statistics-promotion-card')
+);
+
+const extensionStoreDataService = {};
+
+describe('src/module/sw-dashboard/component/sw-dashboard-statistics-promotion-card', () => {
+    async function createWrapper(isAppExistingInTheStore = true, hasPermission = true) {
+        const app = !isAppExistingInTheStore ? null : {
+            id: 99999,
+            label: 'Statistics service app by shopware',
+            name: STATISTICS_APP_NAME
+        };
+
+        extensionStoreDataService.getExtensionByName = hasPermission
+            ? jest.fn(() => Promise.resolve(app))
+            : jest.fn(() => Promise.reject(new Error('Request failed with status code 403')));
+
+        const wrapper = await mount(await Shopware.Component.build('sw-dashboard-statistics-promotion-card'), {
+            global: {
+                provide: {
+                    extensionStoreDataService,
+                    acl: {
+                        can: () => hasPermission
+                    }
+                },
+                stubs: {
+                    'sw-button': await wrapTestComponent('sw-button'),
+                    'sw-button-deprecated': await wrapTestComponent('sw-button-deprecated', { sync: true }),
+                    'sw-icon': true
+                }
+            }
+        });
+
+        wrapper.vm.$router = {
```

Comment:
Shouldn't the router be mocked with this?

https://test-utils.vuejs.org/guide/advanced/vue-router.html#Using-a-Mocked-Router